tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -1109,8 +1113,35 @@ test.describe( 'Nested Tabs tests @nested-tabs', () => {
 		await page.locator( '.eicon-video-camera' ).first().click();
 		await page.locator( '.elementor-control-background_video_link input' ).fill( videoUrl );
 
+		await editor.selectElement( contentContainerTwoId );
+		await editor.activatePanelTab( 'style' );
+		await page.locator( '.eicon-video-camera' ).first().click();
+		await page.locator( '.elementor-control-background_video_link input' ).fill( videoUrl );
+
+		await editor.selectElement( contentContainerThreeId );
+		await editor.activatePanelTab( 'style' );
+		await page.locator( '.eicon-video-camera' ).first().click();
+		await page.locator( '.elementor-control-background_video_link input' ).fill( videoUrl );
+
 		await expect( contentContainerOne ).toHaveAttribute( 'data-model-cid', firstTabContainerModelCId );
 		await expect( videoContainer ).toHaveCount( 1 );
+
+		await page.waitForTimeout( 3000 );
```

Comment:
`waitForTimeout` is not the best practice but sometimes there are no other options.  
What `3000ms` means?
If we are using it in several places we can set it as a variable.

Reply:
@davseve Ideally I would wait for the iframe width value to be updated. Do you have an idea how we could wait for that value to be updated inside PlayWright?

Reply:
I don't have a good suggestion, maybe using a `mutationObserver` but it will be an overkill